apps/e2e/ag-grid-storybook/src/app/ag-grid/ag-grid-stories.component.ts

```diff
@@ -1,79 +1,255 @@
+                      headerName: '',
+                      sortable: false,
+                      cellRenderer: ContextMenuComponent,
+                      maxWidth: 55,
+                    },
+                  ]
+                : [])(),
+            ...columnDefinitions
+              .slice(0, 10)
+              .filter(
+                (col) => col.field !== 'birthday' || dataSet.id !== 'validation'
+              )
+              .map((colDef) => {
+                const additionalType =
+                  colDef.field === 'seasons_played' &&
+                  dataSet.id === 'validation'
+                    ? [SkyCellType.NumberValidator]
+                    : [];
+                const cellRendererParams =
+                  colDef.field === 'seasons_played' &&
+                  dataSet.id === 'validation'
+                    ? {
+                        skyComponentProperties: {
+                          validator: (value: unknown) =>
+                            !!value && value < 18 && value > 0,
+                          validatorMessage:
+                            'Expected a number between 1 and 18.',
+                        },
+                      }
+                    : undefined;
+                return {
+                  ...colDef,
+                  type: [
+                    ...(Array.isArray(colDef.type)
+                      ? colDef.type
+                      : [colDef.type]),
+                    ...additionalType,
+                  ].filter((type) => !!type),
+                  cellRendererParams,
+                };
+              }),
+          ],
+          context: {
+            rowDeleteIds: [],
+          },
+          suppressColumnVirtualisation: true,
+          suppressHorizontalScroll: true,
+          suppressRowVirtualisation: true,
+          onGridReady: (params) => {
+            if (dataSet.id === 'row-delete') {
+              params.api.addEventListener(
+                RowNode.EVENT_ROW_SELECTED,
+                ($event: RowSelectedEvent) => {
+                  if ($event.node.isSelected()) {
+                    this.rowDeleteIds = this.rowDeleteIds.concat([
+                      $event.node.id,
+                    ]);
+                  } else {
+                    this.rowDeleteIds = this.rowDeleteIds.filter(
+                      (id) => id !== $event.node.id
+                    );
+                  }
+                }
+              );
+            }
+
+            params.api.addEventListener(
+              Events.EVENT_FIRST_DATA_RENDERED,
+              () => {
+                (
+                  this.#gridsReady.get(dataSet.id) as BehaviorSubject<boolean>
+                ).next(true);
+              }
+            );
+          },
+          rowData: dataSet.data,
+        },
+      });
+    });
   }
 
-  @Input()
-  public domLayout: 'normal' | 'autoHeight' | 'print' = 'autoHeight';
+  public ngAfterViewInit(): void {
```

Comment:
Delay the Cypress test until 

* the theme service loaded
* all grids have loaded data
* (wait 1 second for any async grid tasks)
* scroll the `back-to-top` grid
* select a row to delete
* trigger a validation popover